crates/nargo/src/cli/fs/keys.rs

```diff
@@ -31,9 +31,9 @@ pub(crate) fn fetch_pk_and_vk<P: AsRef<Path>>(
     prove_circuit: bool,
     check_proof: bool,
 ) -> Result<(Vec<u8>, Vec<u8>), CliError> {
-    let mut acir_hash_path = PathBuf::new();
-    acir_hash_path.push(circuit_build_path.as_ref());
-    acir_hash_path.set_extension(ACIR_EXT.to_owned() + ".sha256");
+    let mut acir_hash_path = PathBuf::from(circuit_build_path.as_ref());
+    acir_hash_path.set_extension("json.sha256");
```

Comment:
Not a blocker as its unrelated; an extension of .checksum would be better here as we don't care for it being sha256